src/execCmd.ts

```diff
@@ -303,3 +303,159 @@ export function execCmd<T = Collection>(
     }
   }
 }
+
+function toString(arrOrString: Many<string>): string {
+  if (Array.isArray(arrOrString)) {
+    return arrOrString.join('');
+  }
+  return arrOrString;
+}
+
+function toArray(arrOrString: Many<string>): string[] {
```

Comment:
use `ensureArray` from kit

Reply:
`ensureArray` throws an error if the provided value isn't an array, which isn't what I'm doing here